e2e/pages/home.page.js

```diff
@@ -0,0 +1,10 @@
+var HomePage = function () {
+    
+    this.greetingHeader = $('main.container h1');
```

Comment:
I think it would be better to search elements by id or class name rather than tag.